optimum/gptq/quantizer.py

```diff
@@ -182,13 +214,49 @@ def __init__(
                 )
         self.exllama_version = self.exllama_config["version"]
 
+    def select_quant_linear(self, device_map: Union[str, dict]):
+        if is_gptqmodel_available():
+            self.quant_linear = hf_select_quant_linear(
+                bits=self.bits,
+                group_size=self.group_size,
+                desc_act=self.desc_act,
+                sym=self.sym,
+                checkpoint_format=self.checkpoint_format,
+                meta=self.meta,
+                device_map=device_map,
+                backend=self.backend,
+            )
```

Comment:
@SunMarc  GPTQModel is exposing `hf_` prefixed stable apis to transfomer/peft/optimum that will not change over time. So any calls to GPTQModel will be `hf_` prefixed. 

Here our quant linear selection requiring the full knowledge of `sym`, `checkpoint_format`, `meta`, `device_map`, and `backend` before deciding on the correct quant_linear to use.